server.rb

```diff
@@ -1,18 +1,71 @@
 require 'rubygems'
 require 'bundler/setup'
+require 'socket'
 require 'rack'
 
 class Server
+
   def startup
     puts 'starting the server'
     app = Proc.new do |env|
-      ['200', {'Content-Type' => 'text/html'}, ["App has started. The time is #{Time.now.strftime("%I:%M:%S %p")}"]]
+      ['200', {'Content-Type' => 'text/html'}, ["App has started. The time is #{Time.now.strftime('%I:%M:%S %p')}"]]
     end
 
+    # TODO create a UDP echo server and start it
+
+    TCPEchoServer.new(8012).start
```

Comment:
is there a reason to be using port 8012 specifically? I setup aws to use port 3030.